.github/workflows/netlify-deploy.yml

```
@@ -0,0 +1,80 @@
+        env:
+          GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
+        run: flutter build web --dart-define=GOOGLE_CLIENT_ID="$GOOGLE_CLIENT_ID"
+
+      - name: Install Netlify CLI
+        if: steps.check.outputs.configured == 'true'
+        run: npm install -g netlify-cli@17
+
+      - name: Deploy preview (pull request)
+        if: steps.check.outputs.configured == 'true' && github.event_name == 'pull_request'
+        working-directory: packages/credential_manager/example
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)